Networking/Networking/Model/Product/ProductVariation.swift

```diff
@@ -188,14 +188,18 @@ public struct ProductVariation: Codable, GeneratedCopiable, Equatable {
 
         // Even though the API docs claim `manageStock` is a bool, it's possible that `"parent"`
         // could be returned as well (typically with variations) — we need to account for this.
-        // A "parent" value means that stock mgmt is turned on + managed at the parent product-level, therefore
-        // we need to set this var as `true` in this situation.
+        // A "parent" value means that stock mgmt is turned off at the product variation and it is managed at the parent product-level.
+        // Therefore, we need to set this var as `false` in this situation.
         // See: https://github.com/woocommerce/woocommerce-ios/issues/884 for more deets
         let manageStock = container.failsafeDecodeIfPresent(targetType: Bool.self,
                                                             forKey: .manageStock,
                                                             alternativeTypes: [
                                                                 .string(transform: { value in
-                                                                    value.lowercased() == Values.manageStockParent ? true : false
+                                                                    guard value.lowercased() == Values.manageStockParent else {
+                                                                        assertionFailure("Unexpected manage stock value: \(value)")
```

Comment:
Right now we don't have a way to log events on Sentry in the Networking layer yet since it depends on the Tracks dependency (it's a TODO https://github.com/woocommerce/woocommerce-ios/issues/2372), but lemme update this to a `DDLogError` so at least it can be logged when a user writes to support (it's recorded in the application log)

Reply:
Updated here: https://github.com/woocommerce/woocommerce-ios/pull/2588/commits/e1c0277ca3fdd3f048bfe1ae73e72652cd296a23